docs/network.md

```diff
@@ -0,0 +1,220 @@
+# 4. Code flow - finding first peer to connect to.
+Each node maintains list of peers.
+
+New node will get an initial list of peers from:
+- Genesis config `network.boot_nodes`
+- Command line argument `--boot_nodes`
+
```

Comment:
I'd also mention that after connecting to the first node, we'll get the routing table from it - and then we'll do additional connections (up to 40?) with random peers from that list.